src/hsm/PSFCorr.cpp

```diff
@@ -1881,6 +1881,54 @@ namespace hsm {
         float sky_var, const char* shear_est, const std::string& recompute_flux,
         double guess_sig_gal, double guess_sig_PSF, double precision,
         galsim::Position<double> guess_centroid, boost::shared_ptr<HSMParams> hsmparams);
+    template CppShapeData EstimateShearView(
+        const BaseImage<uint16_t>& gal_image, const BaseImage<int>& PSF_image,
+        const BaseImage<int>& gal_mask_image,
+        float sky_var, const char* shear_est, const std::string& recompute_flux,
+        double guess_sig_gal, double guess_sig_PSF, double precision,
+        galsim::Position<double> guess_centroid, boost::shared_ptr<HSMParams> hsmparams);
```

Comment:
I'm pretty sure none of these can possibly get called currently.  There is code in hsm.py to force the mask image to be an ImageI. So none of these UI, US variants can ever be called.

Honestly I think I'd prefer just forcing the gal and psf images into either float or double if they come in as integer images and having the only four instantiations be DDI, FFI, DFI, FDI, and getting rid of the III instantiation that was already here (along with all these new ones).

The other option is to make sure we have all possible combinations, which would need to be ntype^3 (ie. many more than are currently listed here), which seems unwieldy to say the least.